sql/telemetry/firefox_nondesktop_day_2_7_activation_v1/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.firefox_nondesktop_day_2_7_activation_v1`
```

Comment:
We recommend tables always have a version suffix and that views _don't_, so I'd recommend removing the `_v1` from this folder name and the view name here. Explanation in https://github.com/mozilla/bigquery-etl#views